src/main/java/umk/mat/pajda/ProjektZespolowy/misc/ReviewArchiveConverter.java

```diff
@@ -0,0 +1,53 @@
+package umk.mat.pajda.ProjektZespolowy.misc;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import umk.mat.pajda.ProjektZespolowy.DTO.ReviewArchiveDTO;
+import umk.mat.pajda.ProjektZespolowy.DTO.ReviewDTO;
+import umk.mat.pajda.ProjektZespolowy.entity.Review;
+import umk.mat.pajda.ProjektZespolowy.entity.ReviewArchive;
+import umk.mat.pajda.ProjektZespolowy.entity.User;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class ReviewArchiveConverter {
+    private final UserConverter userConverter;
+
+    @Autowired
+    public ReviewArchiveConverter(UserConverter userConverter){
+        this.userConverter = userConverter;
+    }
+
+    public ReviewArchiveDTO createDTO(ReviewArchive source) {
+        ReviewArchiveDTO res = new ReviewArchiveDTO();
+        res.setId(source.getId());
+        res.setRating(source.getRating());
+        res.setComment(source.getComment());
+        // DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+        res.setReviewTimeStamp(source.getReviewTimeStamp());
+        res.setUser(userConverter.createDTO(source.getUser()));
+        res.setClientName(source.getClientName());
+        res.setHashRevID(source.getHashRevID());
+        return res;
+    }
+
+    public ReviewArchive createEntity(ReviewArchiveDTO source) {
+        ReviewArchive res = new ReviewArchive();
```

Comment:
what does `res` mean? Can the name be more meaningful?

Reply:
I will change that